tsconfig.json

```diff
@@ -5,5 +5,8 @@
     "packages/**/src",
     "packages/modular-scripts/types/",
     "packages/create-modular-react-app/template/"
-  ]
+  ],
+  "compilerOptions": {
+    "typeRoots": ["./node_modules/@types", "./packages/modular-scripts/@types"]
```

Comment:
not sure why, but currently, the the types i put in modular scripts aren't being resolved WIP